specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1120 @@
+# Knative Control Plane Contract
+## Abstract
+
+The Knative Eventing platform provides common primitives for routing CloudEvents
+between cooperating HTTP clients. This document describes the structure,
+lifecycle, and management of Knative Eventing resources in the context of the
+[Kubernetes Resource Model](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/resource-management.md).
+An understanding of the Kubernetes API interface and the capabilities of
+[Kubernetes Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+is assumed.
+
+This document does not define the
+[data plane event delivery contract](./data-plane.md) (though it does describe
+how event delivery is configured). This document also does not prescribe
+specific implementations of supporting services such as access control,
+observability, or resource management.
+
+## Background
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+There is no formal specification of the Kubernetes API and Resource Model. This
+document assumes Kubernetes 1.21 behavior; this behavior will typically be
+supported by many future Kubernetes versions. Additionally, this document may
+reference specific core Kubernetes resources; these references may be
+illustrative (i.e. _an implementation on Kubernetes_) or descriptive (i.e. _this
+Kubernetes resource MUST be exposed_). References to these core Kubernetes
+resources will be annotated as either illustrative or descriptive.
+
+This document considers two users of a given Knative Eventing environment, and
+is particularly concerned with the expectations of developers (and language and
+tooling developers, by extension) deploying applications to the environment.
+
+- **Developers** configure Knative resources to implement an event-routing
+  architecture.
+- **Operators** (also known as **platform providers**) provision the underlying
+  event routing resources and manage the software configuration of Knative
+  Eventing and the underlying abstractions.
+
+## RBAC Profile
```

Comment:
I wonder if this is better suited for the conformance test suite docs? 
Instead, it seems like the spec should simply state which resources are REQUIRED vs OPTIONAL to support w/o getting into the details of how to make it happen.